tools/ci_build/github/azure-pipelines/linux-ci-pipeline.yml

```diff
@@ -144,3 +144,39 @@ jobs:
   - task: mspremier.PostBuildCleanup.PostBuildCleanup-task.PostBuildCleanup@3
     displayName: 'Clean Agent Directories'
     condition: always()
+
+- job: Linux_TVM_CI
+  timeoutInMinutes: 90
+  workspace:
+    clean: all
+  pool: Onnxruntime-MMS-Ubuntu-2004-CPU
+  steps:
+  - checkout: self
+    clean: true
+    submodules: recursive
+    
+  - task: UsePythonVersion@0
+    inputs:
+      versionSpec: '3.8'
+      addToPath: true
+
+  - task: CmdLine@2
+    inputs:
+      script: |
```

Comment:
Maybe we should add installation of standard dependencies for TVM: 
```Shell
sudo apt-get update &&
sudo apt-get install -y libtinfo-dev zlib1g-dev build-essential libedit-dev libxml2-dev
```
A more detailed list of dependencies can be found in section **Build the Shared Library** of the official [documentation](https://tvm.apache.org/docs/install/from_source.html).